inst/qml/doeFactorial.qml

```diff
@@ -38,6 +38,14 @@ Form
 			max:								256
 		}
 
+        IntegerField
+        {
+            visible:                            false
+            id:                                 numberOfFactorsForTable
+            name:                               "numberOfFactorsForTable"
+            defaultValue:                       numberOfFactors.value
+        }
+
```

Comment:
please use tabs for indents and not spaces in QML...

Reply:
I think there might be some conversion issues or something, because I know that I'm using tabs and not spaces in QML

Reply:
Do you know if there's some option in Qt Creator that could be at fault? Turning tabs into spaces?